build_collection.sh

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env bash
 
+VERSION_RELEASE=` echo ${GITHUB_REF##*/} | grep -oP '(?<=Version_)[^ ]*'`
+
+sed -i "s/\$RELEASE/${VERSION_RELEASE}/g" galaxy.yml
```

Comment:
If `GITHUB_REF` variable is not defined (for example when manually executed), the `galaxy.yml` file will become corrupt. Should this be a required parameter for `build_collection.sh`?

Reply:
I configured build_collection to work with a parameter for the version.